tests/components/pi_hole/test_init.py

```diff
@@ -156,4 +137,4 @@ async def test_enable_service_call(hass):
 
         await hass.async_block_till_done()
 
-        assert mock_enable.call_count == 2
+        assert mocked_hole.enable.call_count == 2
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)